terraform/product/main.tf

```diff
@@ -0,0 +1,167 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+data "juju_model" "opencti" {
+  name = var.model
+}
+
+data "juju_model" "opencti_db" {
+  name = var.db_model
+
+  provider = juju.opencti_db
+}
+
+module "opencti" {
+  source      = "../charm"
+  app_name    = var.opencti.app_name
+  channel     = var.opencti.channel
+  config      = var.opencti.config
+  constraints = var.opencti.constraints
+  model       = data.juju_model.opencti.name
+  revision    = var.opencti.revision
+  base        = var.opencti.base
+  units       = var.opencti.units
+}
+
+module "opensearch" {
+  source      = "git::https://github.com/weiiwang01/opensearch-operator//terraform/charm/simple_deployment?ref=2/edge"
```

Comment:
Just out of curiosity, should this point to a Canonical repository? Or we could vendor this into the modules here as well perhaps?